src/components/o-testimonials/js/testimonials.js

```diff
@@ -0,0 +1,163 @@
+import Swipe from '../../../js/swipe';
+import on from '../../../js/on';
+import UiEvents, { EVENTS } from '../../../js/ui-events';
+
+class Testimonials extends UiEvents {
+  static DEFAULTS = {
+    controlLeft: '.js-o-testimonials__control-left',
+    controlRight: '.js-o-testimonials__control-right',
+    slides: '.js-o-testimonial__item',
+    slider: '.js-o-testimonials',
+  };
+
+  constructor(wcNode, options = {
+  }) {
+    // eslint-disable-next-line no-param-reassign
+    options = {
+      ...Testimonials.DEFAULTS,
+      ...options,
+    };
+
+    super(wcNode, options);
+
+    this.slideIndex = 0;
+    this.autoRotateEnabled = true;
+    this.autoRotateTimeInMiliseconds = 5000;
+    this.options = options;
+    this.wcNode = wcNode;
+
+    this.init();
+  }
+
+  init() {
+    this.slides = this.wcNode.querySelectorAll(this.options.slides);
+    this.controlLeft = this.wcNode.querySelector(this.options.controlLeft);
+    this.controlRight = this.wcNode.querySelector(this.options.controlRight);
+    this.slider = this.wcNode.querySelector(this.options.slider);
+    this.autoRotateTimeInMiliseconds = this.slider.getAttribute('data-auto-rotate-time');
+    this.autoRotateEnabled = this.slider.getAttribute('data-auto-rotate-enabled');
+    this.hideAllSlides();
+    if (this.slides.length < 2) {
```

Comment:
check ESLint errors here, should be missing newline

Reply:
or check travis